cpp/util/etcd.cc

```diff
@@ -472,9 +472,18 @@ void EtcdClient::WatchInitialGetDone(WatchState* state, GetResponse* resp,
   }
 
   // TODO(pphaneuf): Need better error handling here. Have to review
-  // what the possible errors are, most of them should probably be
-  // dealt with using retries?
-  CHECK(task->status().ok()) << "initial get error: " << task->status();
+  // what the possible errors are, for now we'll just retry after a delay.
+  if (!task->status().ok()) {
+    LOG(WARNING) << "Initial get error: " << task->status();
+    LOG(INFO) << "Will retry initial get in "
```

Comment:
Any worry about a WARNING here, as well? For disk flush reasons, I mean? I forget which executor this is on... :stuck_out_tongue: